examples/experimental/dagster-airlift/dagster_airlift/mwaa/auth.py

```diff
@@ -30,8 +30,38 @@ def get_session_info(mwaa: Any, env_name: str) -> Tuple[str, str]:
 
 
 class MwaaSessionAuthBackend(AirflowAuthBackend):
-    def __init__(self, mwaa_client: Any, env_name: str) -> None:
-        self.mwaa_client = mwaa_client
+    """A :py:class:`dagster_airlift.core.AirflowAuthBackend` that authenticates to AWS MWAA.
+
+    Under the hood, this class uses the MWAA boto3 client to request a web login token and then
+    uses the token to authenticate to the MWAA web server.
+
+    Args:
+        mwaa_session (Any): The boto3 MWAA client
```

Comment:
Is there no way to provide a more specific type than Any here? Also should this be `The boto3 MWAA session` rather than "client" (same thing for the docstring above)

Reply:
Yea - narrowed to boto3.Session.